### ISSUE TITLE
Engine tab doesn't work as expected.

### ISSUE THREAD

**Issue by dalf** [searxng/searx-space#21]
#### Description
All instances except one display 🟡 or ?.

#### Current way to have ✔️ and ❌
if your searx setup use docker, then see 
* https://github.com/searx/searx-docker
* more precisely https://github.com/searx/searx-docker/blob/e4e3aa48f448c79873897fa216dd78cda9a43011/docker-compose.yaml#L91-L101
* https://github.com/searx/searx-checker
* https://hub.docker.com/r/searx/searx-checker

There is no documentation on how to install searx-checker if you don't use docker.
Basically you need to call every day (cron.daily) :
```sh
python3 checker/checker.py -o <somewhere>/status.json http://127.0.0.1:8888
````
Then map ```<your searx instance url>/status``` to ```<somewhere>/status.json```

Example: https://a.searx.space/status

#### searx-checker should be embedded into searx
* https://github.com/asciimoo/searx/issues/1559

**Comment by HLFH** [comment]
Besides the `cron` call, I have added the necessary documentation for searx-checker without Docker in this PR: https://github.com/searx/searx-checker/pull/8

**Comment by vojkovic** [comment]
Tab removed
